src/docs/user/references/VariablesReference.adoc

```diff
@@ -1,140 +1,183 @@
 This section describes useful variables, their scope, and their usage.
 
-[cols="1,1,1,1", options="header"]
+[cols="1,1,1,1,1", options="header"]
 
 |===
 
 | Variable name and description
 | Use from a script
 | Use from a native task
 | Use from the workflow
+| Available in
 
-| *Task result*. See <<_passing_data_between_tasks_using_variables>>.
+| *Task result*. Variable set by the user to return the result of a task. See <<_passing_data_between_tasks_using_variables>>.
```

Comment:
If I remember correctly, it is a Java object from our API.  I already had support questions related to the type of this variable at least 3 times (but also to be consistent to what is said for the fork environment variable) it would be nice to give a bit more information.